ament_copyright/ament_copyright/__init__.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Actually, Tully's indention sort of makes more sense, I'm not sure where we got the indention for the other files (I was comparing against `ament_tools`'s files). His is 5 spaces off the `#`, but there is one standard ` ` after the `#`, then the standard 4 space indention for the URL to be in a sort of "pre" box. Also the `LICENSE` file (which is where this snippet came from) has the URL indented four spaces from the start of the previous content.